backend/src/ee/routes/v1/access-approval-policy-router.ts

```diff
@@ -11,21 +12,18 @@ export const registerAccessApprovalPolicyRouter = async (server: FastifyZodProvi
     url: "/",
     method: "POST",
     schema: {
-      body: z
-        .object({
-          projectSlug: z.string().trim(),
-          name: z.string().optional(),
-          secretPath: z.string().trim().default("/"),
-          environment: z.string(),
-          approvers: z.string().array().default([]),
-          groupApprovers: z.string().array().default([]),
-          approvals: z.number().min(1).default(1),
-          enforcementLevel: z.nativeEnum(EnforcementLevel).default(EnforcementLevel.Hard)
-        })
-        .refine((data) => data.approvers.length > 0 || data.groupApprovers.length > 0, {
-          path: ["approvers", "groupApprovers"],
-          message: "At least one approver should be provided."
-        }),
+      body: z.object({
+        projectSlug: z.string().trim(),
+        name: z.string().optional(),
+        secretPath: z.string().trim().default("/"),
+        environment: z.string(),
+        approvers: z
+          .object({ type: z.nativeEnum(ApproverType), id: z.string() })
+          .array()
+          .min(1),
```

Comment:
I would add a custom message on the approvers array.

I forget the exact syntax, but I think it's just a separate parameter like so: `.min(1, { message: "At least one approver should be provided" } )`